src/test/java/sorinfoca/Tests/Tema6/DemoQaWindow.java

```diff
@@ -1,4 +1,4 @@
-package sorinfoca.Tests.Tema6;
+package andreeaG.sorinfoca.Tests.Tema6;
```

Comment:
It seems like you moved sorinfoca package inside andreeaG package. If you can revert this change too. If not, we'll take it in today's meeting. Thanks!